README.md

```diff
@@ -28,3 +28,23 @@ can be started by running the following command from the root of this project:
 Docker container.
 
 ---
+
+## FSM vs StateMachine
+FSM provides a dedicated task that all FSMs share. FSM events are submitted
+to a queue and processed directly by the FSM task, which move the FSM
+between diferrent state. This can be nice to have when the software has
+many FSMs that need to be used but doesn't need tight control of the
+run loop. It can save memory by having all events processed in a single
+place and dispatched the the approriate FSM.
```

Comment:
I think this is meant to say "dispatch the appropriate FSM."

Reply:
It should read "dispatched to the appropriate FSM".